Docs: adding documentation for the new time zone functionality (#24080)
grafana/grafana: docs/sources/reference/timerange.md

Grafana offers the ability to override the `now` value on a per dashboard basis. Most commonly, this feature is used to accommodate known delays in data aggregation to avoid null values.

### Time zone options
Starting in version `7` of Grafana users have the ability to override the time zone used to display date and time values in a dashboard. Apart from the standard [ISO 8601 time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) users also can select the following options:

> 1. **Default**: The default selected time zone for the user profile or organization will be used. If no time zone is specified for the user profile or the organization `Local browser time` will be used.

Author: Rorget that! 🙌

2. **Local browser time**: The time zone configured for the viewing user browser will be used. This is usually the same time zone as set on the compture.

> With this feature, you can specify the local time zone of the service or system that you are monitoring. This can be helpful when monitoring a system or service that operates across several time zones.

Reviewer: I recommend moving this paragraph up to the intro, under line 58. This is very helpful for users trying to figure out if this feature is what they are looking for.

Grafana offers the ability to override the `now` value on a per dashboard basis. Most commonly, this feature is used to accommodate known delays in data aggregation to avoid null values.

### Time zone options

> Starting in version `7` of Grafana users have the ability to override the time zone used to display date and time values in a dashboard. 

Reviewer: ```suggestion
Starting in Grafana 7.0, you can override the time zone used to display date and time values in a dashboard. 
```

With this feature, you can specify the local time zone of the service or system that you are monitoring. This can be helpful when monitoring a system or service that operates across several time zones.

> Apart from the standard [ISO 8601 time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) users also can select the following options:

Reviewer: ```suggestion
Apart from the standard [ISO 8601 time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) you can select the following options:
```

1. **Default**: The default selected time zone for the user profile or organization will be used. If no time zone is specified for the user profile or the organization `Local browser time` will be used.
2. **Local browser time**: The time zone configured for the viewing user browser will be used. This is usually the same time zone as set on the compture.

> One of the benefits with this feature is that you can specify the local time zone of the service/system that you are monitoring which can be really helpful when monitoring a system/service that operates across several time zones.

Reviewer: ```suggestion
With this feature, you can specify the local time zone of the service or system that you are monitoring. This can be helpful when monitoring a system or service that operates across several time zones.
```
